code/modules/mob/living/carbon/human/species_types/mothmen.dm

```diff
@@ -16,12 +16,11 @@
 	toxic_food = MEAT | RAW
 	mutanteyes = /obj/item/organ/eyes/moth
 
-/datum/species/moth/on_species_gain(mob/living/carbon/C)
+/datum/species/moth/regenerate_organs(mob/living/carbon/C,datum/species/old_species,replace_current=TRUE)
```

Comment:
Seems odd that an adminheal should access client prefs, are there issues with clone damage deleting the `"moth_wings"` feature from the DNA or something?

Reply:
Issue I was seeing was that wings are burned off by handle_fire() @ l37, which changes the state without deleting them outright.

Admin healing the mob afterwards doesn't restore them to their undamaged state, and since regenerate_organs is called in on_species_gain() anyway, this seemed to be the cleanest way to resolve the issue without having to duplicate the code.

Cloning shouldn't be having any issue since growclone() just spawns a new human mob and applies the species again.